docker-for-mac/faqs.md

```diff
@@ -211,7 +211,7 @@ Starting with Docker for Mac Beta 27 and Stable 1.12.3, all trusted certificate
 authorities (CAs) (root or intermediate) are supported.
 
 For full information on adding server and client side certs, see [Adding
-TLS certificates](index.md#adding-tls-certificates) in
+TLS certificates](index.md#add-tls-certificates) in
```

Comment:
@joaofnfernandes , the links are good but maybe change the name of the link to match the anchor (from Adding to Add)?